vcsutils/webhookparser/webhookparser.go

```diff
@@ -45,6 +45,34 @@ type WebhookInfo struct {
 	Author WebHookInfoUser `json:"author,omitempty"`
 	// CompareUrl is HTML URL to see git comparison between commits (Push event only)
 	CompareUrl string `json:"compare_url,omitempty"`
+	// PullRequest encapsulates information of the pull request.
+	PullRequest *PullRequestInfo `json:"pull_request,omitempty"`
+}
+
+// PullRequestInfo contains information about pull request.
+type PullRequestInfo struct {
```

Comment:
There is PullRequestInfo object in vcsclient.go. I find it a little confusing.
```suggestion
type PullRequestWebhookInfo struct {
```